### PR TITLE
Add a note about what to comment on during review

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Add a note about what to comment on during review including things like too many changes

### CODE REVIEW

**Comment by EladNLG** [comment]
Basically, a thing should be approved if you were willing to work on it yourself.

**Comment by GeckoEidechse** [comment]
> Basically, a thing should be approved if you were willing to work on it yourself.

I can add that line but I can assure you that this will significantly reduce the number of approved PRs including from me, as it will prevent me from approving a PR that tested correctly simply cause I'm not willing to work on code that I'm not fully able to understand.
Previously I'd approve PR that tested correctly even though I would not be willing to maintain them.

**Comment by EladNLG** [comment]
> I can add that line but I can assure you that this will significantly reduce the number of approved PRs including from me, as it will prevent me from approving a PR that tested correctly simply cause I'm not willing to work on code that I'm not fully able to understand. Previously I'd approve PR that tested correctly even though I would not be willing to maintain them.

If you merged something that you weren't meaning to maintain - you would only make the codebase worse by adding something that will only gather dust - and then when a bug pops up you'd need to relearn the entire PR's code just to fix it, which would slow down work on other things (and as a result, other PRs)

That's my exact intention with this change.

**Comment by EladNLG** [comment]
Of course, this doesn't mean every PR should be closed right out of the gate - there should be a reason why you aren't willing to maintain this.

If the code isn't understandable, **say jt**.

But if there's nothing you can find that necessarily needs change, then maybe you don't like the feature in the first place.

File: docs/modding-and-development/development/reviewing.md
Comment (EladNLG) [inline docs/modding-and-development/development/reviewing.md]: Spelling mistake >:(

**Review by EladNLG — CHANGES_REQUESTED** [review]
\>:((((

File: docs/modding-and-development/development/reviewing.md
Comment (GeckoEidechse) [reply docs/modding-and-development/development/reviewing.md]: Fixed in 489e966d95bbac4929134624bd8d45f6123d24ad

**Comment by GeckoEidechse** [comment]
> If you merged something that you weren't meaning to maintain - you would only make the codebase worse by adding something that will only gather dust - and then when a bug pops up you'd need to relearn the entire PR's code just to fix it, which would slow down work on other things (and as a result, other PRs)
> 
> That's my exact intention with this change.

Handing off responsibility to the code maintainer kinda goes against the common practice for open-source code maintaineship, where usually it's a joint ownership of the lines of code afterwards. That being said I added the suggested change in afe67ff7b4a17688c429bf76e37f7c7b33e7f1cc.

Do note that if this PR is merged, it'll be essentially my resignation from all code reviews/testing as I'm not willing to approve any PRs that I would have to maintain afterwards.

As such I'm forfeiting merging this PR to someone else.

File: docs/modding-and-development/development/reviewing.md
Comment (Alystrasz) [inline docs/modding-and-development/development/reviewing.md:60]: > You found no issues when looking at the code or during testing.

It should be "[...] at the code *AND* during testing", since I can't imagine reviewers hitting the approve button without actually running the proposed code.

**Review by Alystrasz — CHANGES_REQUESTED** [review]
Some wording stuff

File: docs/modding-and-development/development/reviewing.md
Comment (GeckoEidechse) [reply docs/modding-and-development/development/reviewing.md:60]: > I can't imagine reviewers hitting the approve button without actually running the proposed code.

Oh sweet summer child if only you knew the things I've seen :P

Comment (Alystrasz) [reply docs/modding-and-development/development/reviewing.md:60]: > if only you knew the things I've seen

I for sure hope my research colleagues are not reading my words 🙄

**Comment by GeckoEidechse** [comment]
Closed in favour of better worded PR by @EladNLG

**Pull request closed without merging**
